docker-entrypoint.sh

```diff
@@ -29,6 +29,10 @@ case "$1" in
         exec "$@"
```

Comment:
@jrgm what do you think about either removing this existing `python` entrypoint, or changing it so that it executes `pypy` under the hood? Given the way the container is build I don't think any good can come from trying to run things in this container with `python` rather than `pypy`.

Reply:
Yes, good point. I'll remove the python entrypoint.